image/templates/helm/shared/config-templates/scanner-v4/matcher-config.yaml.tpl

```diff
@@ -23,6 +23,12 @@ matcher:
       {{ if not (kindIs "invalid" ._rox.scannerV4.db.source.maxConns) -}} pool_max_conns={{._rox.scannerV4.db.source.maxConns}} {{- end }}
       client_encoding=UTF8
     password_file: /run/secrets/stackrox.io/secrets/password
+  vulnerability_version:
+  {{- if ._rox.env.centralServices }}
+  vulnerabilities_url: https://central.{{ .Release.Namespace }}.svc/api/extensions/scannerdefinitions?version={{.Version}}
+  {{- else }}
+  vulnerabilities_url: https://sensor.{{ .Release.Namespace }}.svc/api/extensions/scannerdefinitions?version={{.Version}}
+  {{- end }}
```

Comment:
Helm will try to replace `{{.Version}}` the same as it does for `{{.Release.Namespace}}` - will need to escape version so helm ignores if keeping this syntax:

```suggestion
  {{- if ._rox.env.centralServices }}
  vulnerabilities_url: https://central.{{ .Release.Namespace }}.svc/api/extensions/scannerdefinitions?version={{ "{{" }}.Version{{ "}}" }}
  {{- else }}
  vulnerabilities_url: https://sensor.{{ .Release.Namespace }}.svc/api/extensions/scannerdefinitions?version={{ "{{" }}.Version{{ "}}" }}
  {{- end }}
```

Reply:
I wonder if we should keep using ROX_VERSION
What do you think?

Reply:
It would be taking advantage of `text/template` - we need to 'escape it' from helm, because helm also uses the same syntax and would replace it.  

Without the escape, helm would set this in the configmap (assuming no `Version` variable exists, which when I tested is exactly what happened):
```
https://sensor.stackrox.svc/api/extensions/scannerdefinitions?version=
```
With the escape, helm sets this:
```
https://sensor.stackrox.svc/api/extensions/scannerdefinitions?version={{.Version}}
```
Because helm using the same syntax, this may be 'more' confusing to the average user of the product, I'm OK if want to change this back to `ROX_VERSION` and do a string replace.  But confirm with @jvdm first.